recipes/hy/meta.yaml

```diff
@@ -0,0 +1,68 @@
+{% set name = "hy" %}
+{% set version = "0.12.1" %}
+{% set md5 = "552e80e5ed1d32fd34f62ef53f78dbbc" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/46/00/b0a6855a11510b7d1f503ca723cfc2a411b940b9f9744c9b3f469533c92d/{{ name }}-{{ version }}.tar.gz
```

Comment:
Please replace with this [`url`]( https://github.com/conda-forge/staged-recipes/blob/c3147087640b7022d0ab72f374fdb7ec88cf355b/recipes/example/meta.yaml#L18 ).